### ISSUE TITLE
error after !addfriend

### ISSUE THREAD

**Issue by qpeckin** [HerrEurobeat/steam-comment-service-bot#85]
**Describe the bug**
I got the following error after using !addfriend

![I3EKXJKc](https://user-images.githubusercontent.com/64363113/111070720-35c4fc00-84d3-11eb-8487-5af9359e0676.png)

**Comment by HerrEurobeat** [comment]
Thanks for reporting, I will look into this. For anyone needing a temporary fix you can disable the limited check:  
Go into the `bot.js` file, go to line 668, and remove this part:  
```
if (controller.botobject[i].limitations.limited == true) {
   logger(`Can't add friend ${args[0]} with bot${i} because the bot account is limited.`) 
   return;
}
```

**Comment by HerrEurobeat** [comment]
The check will now be ignored if the bot can't check if the bot account is limited. This should be fine.

**Comment by HerrEurobeat** [comment]
Fixed in 9aaea39 (Version 2.10.2).

**Comment by HerrEurobeat** [comment]
Whoops it seem like I made a small mistake which can still cause this error.  
The new check only checks for `botobject[0]` and not for the current iteration (`botobject[i]`).  
This needs to be changed and should resolve the issue completely.

**Comment by HerrEurobeat** [comment]
Fixed in 7ca8ded for real this time (unless I made a stupid mistake again) (Version 2.10.4).
